<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopWidget</class>
 <widget class="QWidget" name="DesktopWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
     border: 1px solid #404040;
     border-radius: 2px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #606060, stop: 1 #404040);
     max-height: 60px;
 }</string>
     </property>
     <property name="text">
      <string>FaceBrick</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="postsListView">
     <property name="styleSheet">
      <string> QPushButton {
     border: 1px solid #404040;
     border-radius: 2px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #606060, stop: 1 #404040);
     max-height: 60px;
 }</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="upButton">
       <property name="styleSheet">
        <string notr="true"> QPushButton {
     border: 1px solid #404040;
     border-radius: 2px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #606060, stop: 1 #404040);
     max-height: 60px;
 }</string>
       </property>
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="styleSheet">
        <string notr="true"> QPushButton {
     border: 1px solid #404040;
     border-radius: 2px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #606060, stop: 1 #404040);
     max-height: 60px;
 }</string>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downButton">
       <property name="styleSheet">
        <string notr="true"> QPushButton {
     border: 1px solid #404040;
     border-radius: 2px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #606060, stop: 1 #404040);
     max-height: 60px;
 }</string>
       </property>
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
